tests/js/auth/oauth/service-worker/listeners.test.ts

```
@@ -0,0 +1,371 @@
+        error: 'No token received'
+      })
+    })
+
+    it('handles errors when storing token', async () => {
+      // Mock successful Google auth token generation
+      mockGetAuthToken.mockImplementation((options, callback) => {
+        callback('google-token')
+      })
+
+      // Mock storage.set to throw error
+      mockStorageSet.mockImplementationOnce((data, callback) => {
```

ESLint

[error] 'data' is defined but never used. Allowed unused args must match /^_$/u.

(@typescript-eslint/no-unused-vars)